stable/weave-cloud/templates/manifest.yaml

```diff
@@ -0,0 +1,402 @@
+apiVersion: v1
+kind: List
+items:
+  - metadata:
+      labels:
+        name: weave-flux
```

Comment:
Is the expectation that this can only be installed once per namespace per cluster?